base/src/Data/Macaw/Discovery.hs

```diff
@@ -501,10 +501,10 @@ defaultClassifier = branchClassifier
                 <|> noreturnCallClassifier
                 <|> callClassifier
                 <|> returnClassifier
-                <|> directJumpClassifier
                 <|> jumpTableClassifier
                 <|> pltStubClassifier
                 <|> tailCallClassifier
+                <|> directJumpClassifier
```

Comment:
If I understand correctly, this will currently yield some false positives when classifying tail calls. It would be worth leaving a comment explaining why this happens and to leave breadcrumbs in case we need to resolve this in the future.